src/controllers/pages.js

```diff
@@ -23,7 +23,13 @@ export const dashboard = async (req, res) => {
 		currentLesson = await Lesson.findById(req.user.currentClass);
 		currentLesson = await getLessonProgress(req.user.id, currentLesson);
 	}
-	res.render("dashboard", { lesson: currentLesson });
+
+	const totalCount = await LessonProgress.countDocuments();
```

Comment:
If traffic ever starts really picking up, for totalCount, it may then be best to do this query only once on server startup, and keep it cached in memory for subsequent requests as it's unlikely the total lessons will change often (if ever). For now though this works for simplicity's sake.